judging-algorithm/templates/rooms_example.csv

```diff
@@ -0,0 +1,4 @@
+name,capacity
+hidden figures,3
+board room,3
+rob’s office,1
```

Comment:
is this used any more? it should just be the 3 rooms, with practically unlimited capacity, since they are big conference rooms